api/filters.go

```diff
@@ -129,6 +130,26 @@ func (api *API) createFilter(w http.ResponseWriter, r *http.Request) {
 	api.respond.JSON(ctx, w, http.StatusCreated, resp)
 }
 
+func (api *API) getFilterDimensions(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	fID := chi.URLParam(r, "id")
+
+	dimensions, err := api.store.GetFilterDimensions(ctx, fID)
+	if err != nil {
+		api.respond.Error(
+			ctx,
+			w,
+			statusCode(err),
+			errors.Wrap(err, "failed to get filter"),
```

Comment:
Message should probably be "failed to get filter dimensions".

Reply:
Consider whether we want the details of the error returned to the user. AS these endpoints will be facing the public exposing details of our database implemetation could open us up to people trying to find exploits. The api package's error type has the ability to return a different message to the user from the error that gets logged like:
```
return Error{
	err: errors.Wrap(err, "failed to get filter dimensions"), // gets logged for our purposes
	message: "failed to get filter dimensions", // singular message gets returned to user, no wrapped error details etc
	logData: log.Data{
	 "id": fID, // example, use log.Data to annotate errors with useful details for our logging
	}
}
```

Reply:
Good point.
I have updated my IDE.
Resolved.

Reply:
Good point as well.